sdk/schemaregistry/azure-schemaregistry/samples/sync_samples/schema_registry.py

```diff
@@ -66,10 +66,10 @@
 SCHEMA_STRING = json.dumps(SCHEMA_JSON, separators=(",", ":"))
 
 
-def register_schema(client, group_name, schema_name, schema_string, format):
+def register_schema(client, group_name, name, schema_string, format):
```

Comment:
since we have updated the parameter naming already, how about we also change `schema_string` to `definition`?